inc/fields/class-shortcode-ui-field-post-select.php

```diff
@@ -137,11 +137,21 @@ public function action_wp_ajax_shortcode_ui_post_field() {
 
 		$query = new WP_Query( $query_args );
 
+		$is_multiple_post_types = count( $query->get( 'post_type' ) ) > 1;
```

Comment:
yes, but if we use 'post_type' => 'any' that means in this query we will get all post types posts. And still the `count( $query->get( 'post_type' ) )` will be 1. And at that time we won't get post type name in the select dropdown. As we are using this condition.
https://codex.wordpress.org/Class_Reference/WP_Query#Type_Parameters